internal/app/wwctl/profile/set/root.go

```diff
@@ -63,6 +66,10 @@ func init() {
 	baseCmd.PersistentFlags().BoolVar(&SetNetDevDel, "netdel", false, "Delete the node's network device")
 	baseCmd.PersistentFlags().BoolVar(&SetNetDevDefault, "netdefault", false, "Set this network to be default")
 
+	baseCmd.PersistentFlags().StringVarP(&SetParam, "param", "p", "", "Define custom parameter")
```

Comment:
Same thing here with regards to `--key/-k`.

Reply:
see `nod/set`